public/app/pipeline-orchestrator.js

```
@@ -500,12 +500,14 @@ class PipelineOrchestrator {
       if (typeof window._vipPreloadModels === 'function') {
-        // Fire-and-forget preload — models load while audio context inits.
-        // Only the canonical four models (matching model-loader.js +
-        // models-manifest.json) are preloaded. Any others would diverge from
-        // the SW Cache and double-download under different filenames.
+        // Fire-and-forget preload of eager models only — silero_vad (~2.3 MB)
+        // and rnnoise (~2 MB) are marked eager:true in models-manifest.json.
+        // demucs_v4 (~87 MB) and bsrnn_vocals (~3.9 MB) are lazy: preloading
+        // them at page init exhausts mobile memory and causes "Aw, Snap!" OOM
+        // crashes before the user has done anything. They are fetched on-demand
+        // when the pipeline actually needs them.
         window._vipPreloadModels(
```

ESLint

[error] 'window' is not defined.

(no-undef)